Engine/Module/ModuleRender.cpp

```diff
@@ -206,8 +218,41 @@ void ModuleRender::GetMeshesToRender(const ComponentCamera *camera)
 	{
 		GetCullingMeshes(App->cameras->main_camera);
 	}
+	SetListOfMeshesToRender(camera);
+}
+void ModuleRender::SetListOfMeshesToRender(const ComponentCamera *camera)
+{
+	opaque_mesh_to_render.clear();
+	transparent_mesh_to_render.clear();
+	float3 camera_pos = camera->camera_frustum.pos;
+	for (int i = 0; i < meshes_to_render.size(); i++)
```

Comment:
For avoiding a warning u should declare i like unsigned int or size_t.

Reply:
cool!